storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2178,6 +2281,18 @@ dberr_t dict_load_tables_from_space_id(space_id_t space_id, THD *thd,
     goto error;
   }
 
+  /* Before 8.0.24 if the table is used in EXCHANGE PARTITION or IMPORT. Even
+  after upgrade to the latest version 8.0.25 (which fixed the duplicate SDI
+  issue), such tables continue to contain duplicate SDI. PXB will scan the DD
+  table "mysql.tables" to determine the correct SDI */
+  if (ib_vector.sdi_vector->m_vec.size() > 2 &&
+      strcmp(space->name, "mysql") != 0 &&
+      fsp_is_file_per_table(space_id, space->flags)) {
+    msg("xtrabackup: duplicate sdi for space %s.\n", space->name);
```

Comment:
Also, suggest user to consider doing OPTIMIZE TABLE on this table
"Please do OPTIMIZE TABLE to rebuild tablespace and the duplicate SDI will be removed"

Can you also please test before adding this comment?

Reply:
On second thought, this will be helpful only for future full backups. incremental backups are not impacted by duplicate SDI, right?

Reply:
incremental backup are also impacted by duplicate sdi. once we rebuild them we can get rid of duplicated sdi in next incremental backup.

Reply:
Ok, can you ajust the message then?